### PR TITLE
feat: add recaptcha to signup screen

### PR DESCRIPTION
**Pull request by davidgamez**
**Summary:**

 Add Google reCaptcha to the Sign-up screen.

Out of scope:
 - reCaptcha component on the CompleteRegistration. This will be done as part of a different PR once #164 is merged.
 
Part of #152 

**Expected behavior:** 

The sign-up screen contains a reCaptcha and is required for submitting a sign-up request. 

Screenshot 2023-11-15 at 4 41 40 PM

**Testing tips:**

- Go to the sign-up screen
- Verify that reCaptcha is on the screen
- Fill in all the required information without clicking the reCaptcha checkbox
- Verify that the reCaptcha error message appears on the screen, _"You must verify you are not a robot."_
- Click the reCaptcha checkbox and click the sign-up button
- Verify that the sign-up request is sent to the screen, and if it is successful, the user is redirected to the accounts page

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `./scripts/api-tests.sh` to make sure you didn't break anything
- [x] Format the title like "feat: [new feature short description]." The title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
- [x] Linked all relevant issues
- [x] Include screenshot(s) showing how this pull request works and fixes the issue(s)

### CODE REVIEW

**Comment by cka-y** [comment]
image
I suggest adding padding between the error message and the captcha component to stay consistent with the other error messages.

File: web-app/src/app/utils/config.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/**
+ * Get environment config
+ * @param key variable key
+ * @returns variable value or empty string
+ */
+export const getEnvConfig = (key: string): string => {
+  return process.env[key] ?? '';
+};
```
Comment (cka-y) [inline web-app/src/app/utils/config.ts:8]: Could you provide guidance on how the key should be setup for local dev?

Comment (davidgamez) [reply web-app/src/app/utils/config.ts:8]: Added a section on the README file on how to provide the values; also updated the outdated script section.

**Comment by davidgamez** [comment]
> image I suggest adding padding between the error message and the captcha component to stay consistent with the other error messages.

I wrapped the reCaptcha component with a Material component so we can use the same margin pattern, here the result
Screenshot 2023-11-16 at 2 09 41 PM

**Review by cka-y — APPROVED** [review]
LGTM

**Pull request merged**